startupscript/post-startup.sh

```diff
@@ -12,13 +12,12 @@ fi
 
 user="$1"
 workDirectory="$2"
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
i get a relative path with that..

this wierd script is based on [this answer ](https://stackoverflow.com/a/3915420)

Reply:
Is a relative path a problem? Often preserving the path used by the caller is most helpful.

But if an absolute path is needed, then a comment indicating that and why would be good.
If we do have `readlink` available, then I'd recommend just using `readlink -f` to get an absolute and canonical path.

Reply:
relative link is not reliably becuase we move around directory in the script. for example, we go to /tmp at line 49. i have hesitation with readlink because this script is executed in the container and we don't have control over the app container image. but i agree with you that it' s more readible.